views/content/token.md

```diff
@@ -306,11 +306,7 @@ The next step is making the buy and sell functions:
 ```
     function buy() payable returns (uint amount){
```

Comment:
would you please keep the return statement, so it matches with function signature?

Reply:
oops that was a terrible mistake :p.